CHANGELOG.rst

```diff
@@ -1,3 +1,16 @@
+pytest-xdist 1.22.5 (2018-07-27)
+================================
+
+Bug Fixes
```

Comment:
there seems to be a `B` missing

Reply:
i suggest to do a minor release adding six

Reply:
OK, done.

About the missing `B`, it is some bug on GH's diff, [the file is fine](https://github.com/nicoddemus/pytest-xdist/blob/release-1.22.5/CHANGELOG.rst).